### ISSUE TITLE
Add a recommended commit naming scheme

### ISSUE THREAD

**Issue by FireIsGood** [OSU-App-Club/beavbright#3]
Currently, commits seem to be split between a few styles.

- Nyumat uses some form of [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Cassius uses imperative tense
- Other commits are not following a format

I propose a new *Contribution Guidelines* suggesting some form (likely Conventional Commits) to new contributors to give a standard&mdash;though not required&mdash;way to write commits. This section could also help write out expectations for contributors (which are currently just [posted on a related blog post](https://www.osuapp.club/blog/spring24-project/)

**Comment by Nyumat** [comment]
Good idea, but not worth it until we start having formal releases and or deploy
